sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/CHANGELOG.md

```diff
@@ -1,14 +1,29 @@
 # Release History
 
-## 1.3.0-beta.3 (Unreleased)
+## 1.3.0-beta.3 (2025-04-10)
 
 ### Features Added
 
-### Breaking Changes
+- Upgraded api-version tag from 'package-2024-11-preview' to 'package-2025-03-preview'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/cb262725d128f6dfec4622cca03bc9e04e2d0f1f/specification/containerregistry/resource-manager/readme.md.
+- Added new struct `RoleAssignmentMode`.
+- Added new class `SourceRegistryCredentials`.
+- Added new attribute `RoleAssignmentMode` in `ContainerRegistryData` and `ContainerRegistryPatch`.
+- Added new attribute `SourceRegistry` in `ContainerRegistryCredentials`.
 
-### Bugs Fixed
+### Breaking Changes
 
-### Other Changes
+- Changed the signature of the following methods in `ContainerRegistryResource`
+    - `ScheduleRun`
+    - `ScheduleRunAsync`
+- Changed the signature of the following methods in `ContainerRegistryRunResource`
+    - `Cancel`
+    - `CancelAsync`
+    - `Update`
+    - `UpdateAsync`
+- Changed the signature of the following methods in `ContainerRegistryTaskResource`
+    - `Update`
+    - `UpdateAsync`
+- Removed the attribute `SourceRegistryLoginMode` from `ContainerRegistryCredentials`
```

Comment:
breaking changes are not allowed unless we bump the major version number, and bumping the major version number is against the versioning strategy in mgmt libraries for .net.
Therefore if we really have breaking changes, please share the information with us about "how is the breaking change and why" so that we could help to find a solution to mitigate them without bumping the major version or breaking existing users

Reply:
Removed breaking changes